TsRandomizer/Archipelago/SlotDataParser.cs

```diff
@@ -25,8 +25,6 @@ public Requirement GetPyramidKeysGate()
 		public static Requirement GetPyramidKeysGate(string pyramidKeysGate)
 		{
 			//TODO: remove when clients & server are update with correct value
-			if (pyramidKeysGate == "GateMilitaryGate")
```

Comment:
we cant remove this yet sorry, We need a fallback as Archipelago sends GateMilitairyGate with AI, so since you renamed it here internally, we still need a fallback from "GateMilitairyGate" to Requirement.GateMilitaryGate

Reply:
Right so okay, I think this if statement was backwards the whole time then? because Archipelago has GateMilitairyGate
https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/timespinner/PyramidKeys.py#L9 , fixing the statement

Reply:
Switched to https://github.com/JarnoWesthof/TsRandomizer/pull/45/commits/8dee614aad171934297aaea90121550c1dcea5fc

Reply:
indeed i also realized it was backwards